plugins/cassandra/cassandrawu.cpp

```diff
@@ -3515,14 +3549,36 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
                 merger->addPostFilters(poorFilters, 0);
                 merger->addPostFilters(remoteWildFilters, 0);
                 const IPostFilter &best = goodFilters.item(0);
+                const char *queryValue = best.queryValue();
+                if (strchr(queryValue, '|'))
+                {
+                    StringArray values;
+                    values.appendList(queryValue, "|");
+                    ForEachItemIn(vidx, values)
+                    {
+                        merger->addResult(*new CassandraResult(fetchDataForKeyWithFilter(best.queryXPath(), values.item(vidx), wuidFilters, sortorder, merger->hasPostFilters() ? 0 : pageSize+startOffset)));
```

Comment:
The dali code excludes empty strings - should this also do that, or should the dali code not bother?